.gitignore

```diff
@@ -27,4 +27,4 @@ AutoDict_*
 *.pdf
 
 # This is a symlink created by setup.sh
-DataFormats/src/classes_def.xml
\ No newline at end of file
+NtupleAnalysis/
```

Comment:
Should just remove NtupleAnalysis from git repo and tracked files rather than add to gitignore